apkid/rules/apk/packers.yara

```diff
@@ -395,3 +451,22 @@ rule yidun : packer
   condition:
     is_apk and (#lib > 1) or ($anti_trick and $entry_point and $jni_func)
 }
+
+
+rule apkpacker : packer
+{
+    meta:
+        description = "ApkPacker"
+        sample      = "087af5aacab8fc8bc7b1dcb7a138c3552d175c74b496056893299bc437422f95"
+        author      = "Eduardo Novella"
+
+    strings:
+        $a = "assets/ApkPacker/apkPackerConfiguration"
+        $b = "assets/ApkPacker/classes.dex"
+        //$c = "assets/config.txt"
```

Comment:
What's the deal with these comments? Any reason to keep them? If so, could you add a comment explaining what they're for

Reply:
Yeah, they are there because we only got a sample, so I don't know if these are malware or packer related. They are there to remember us if we can uncomment them out

Reply:
This should be merged from its branch. Sorry the mess up

Reply:
Could you add a comment to capture what you said here? e.g. "these may be related, but not enough samples to be sure"

Reply:
Done in the other pull-request